src/main/java/com/nike/cerberus/command/core/UpdateStackCommand.java

```diff
@@ -31,8 +31,7 @@
 import java.util.Map;
 
 import static com.nike.cerberus.command.core.UpdateStackCommand.COMMAND_NAME;
-import static com.nike.cerberus.ConfigConstants.SKIP_AMI_TAG_CHECK_ARG;
```

Comment:
What was this used for and what happens now that this is removed?

Reply:
The removal of this does allow an operator to accidentally deploy the wrong AMI and take out CMS. an update CMS command might be nice.